.vagrant/machines/default/virtualbox/action_provision

```diff
@@ -0,0 +1 @@
+1.5:362160bd-c02f-46b1-b552-29ce89c8bb7a
```

Comment:
You should remove all of these files from git and add `.vagrant` to the `.gitignore` file

Reply:
Thanks, this is done.